python/ray/log_monitor.py

```diff
@@ -195,7 +199,7 @@ def check_log_files_and_publish_updates(self):
             # Record the current position in the file.
             file_info.file_position = file_info.file_handle.tell()
 
-            if len(lines_to_publish) > 0 and is_worker:
+            if len(lines_to_publish) > 0:
                 self.redis_client.publish(
```

Comment:
@richardliaw we only publish worker log file, not sure why. Please correct me if I am not right.